client/source/AppHolder.jsx

```diff
@@ -10,11 +10,11 @@ import * as requests from './utilities/axiosRequests.js';
 // MAIN
 const AppHolder = function CreateAppHolder() {
   // STATES
-  const [prodID, setProdID] = useState(40344); // TODO: default view for 'no item searched yet'
+  const [prodID, setProdID] = useState(40356); // TODO: default view for 'no item searched yet'
```

Comment:
Let's change this back to 40344 for merging to main purposes so it's not changing on everyone. 

I do the same thing where I edit it for the product_id I'd like to test, then I change it back.

Reply:
Note on decision to leave as 40356: only one productID with more than 4 questions, and has enough data for all other modules.